### DISCUSSION TITLE
Changing the banner height

### DISCUSSION THREAD

**Discussion by ZuluPilgrim** [GetPublii/Publii#1452] [Help]
Do I use CCS to change the banner height or do I find a default setting somewhere?
![image](https://github.com/GetPublii/Publii/assets/111164245/2122fcd9-6afd-47b3-bc5f-b03057c3da99)

**Comment by ZuluPilgrim** [comment]
Found it.  Its the hero section
![image](https://github.com/GetPublii/Publii/assets/111164245/acfcba4f-6049-46b9-8896-089147da7d0c)

**Reply by 3muchfun** [reply]
I would say it's the Hero Section Height visible in your screenshot....
